tools/react-app-template/src/App.tsx

```diff
@@ -1,82 +1,48 @@
-// (C) 2019-2022 GoodData Corporation
+// (C) 2019-2023 GoodData Corporation
 import React from "react";
 import { BackendProvider, WorkspaceProvider } from "@gooddata/sdk-ui";
 import { Dashboard } from "@gooddata/sdk-ui-dashboard";
 
 import { backend } from "./backend";
-import { workspace } from "./constants";
 import logo from "./assets/gooddata-logo.svg";
-import * as Md from "./md/full";
-
-const Content: React.FC = () => {
-    return (
-        <div className="container">
-            <div className="header">
-                <div className="logo">
-                    <a
-                        className="logo-link"
-                        href="https://sdk.gooddata.com/gooddata-ui"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        <img className="logo-link-image" src={logo} />
-                        <h1 className="logo-link-title">GoodData.UI</h1>
-                    </a>
-                </div>
-                <div className="navigation">
-                    <a
-                        className="navigation-link"
-                        href="https://sdk.gooddata.com/gooddata-ui"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        📚 Official documentation
-                    </a>
-                    <a
-                        className="navigation-link"
-                        href="https://sdk.gooddata.com/gooddata-ui/docs/quickstart.html"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        🚀 Getting started
-                    </a>
-                    <a
-                        className="navigation-link"
-                        href="https://gdui-examples.herokuapp.com/"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        📊 Examples Gallery
-                    </a>
-                    <a
-                        className="navigation-link"
-                        href="https://sdk.gooddata.com/gooddata-ui-apidocs/"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        ⚙️ API reference
-                    </a>
-                </div>
-            </div>
-            <div className="content">
-                <Dashboard dashboard={Md.Dashboards.DashboardEmbedding} />
-            </div>
-            <div className="footer">
-                Powered by{" "}
-                <a href="https://sdk.gooddata.com/gooddata-ui" target="_blank" rel="noopener noreferrer">
-                    GoodData.UI
-                </a>
-            </div>
-        </div>
-    );
-};
+import * as cat from "./catalog";
```

Comment:
In all our examples, we are using `* as Md`, we should probably make it consistent

Reply:
I think `catalog` makes more sense, as our customers will probably not understand `md` as `metadata`. But you're right, we should update it everywhere in this case and such decision should be made by docs team when they start working on docs this quarter. I'll update the code shortly.